.circleci/config.yml

```diff
@@ -89,36 +89,23 @@ jobs:
 #            go get github.com/tcnksm/ghr
 #            ghr -t ${GITHUB_API_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -replace ${CIRCLE_TAG} ./dist/
 
-  build-docker-image:
+  build-and-publish-docker-image:
     executor: docker-publisher
     steps:
       - checkout
       - setup_remote_docker
       - run:
-          name: Build Docker image
-          command: docker build -t $IMAGE_NAME:latest .
+          name: Login to Docker Hub
+          command: 'echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin'
       - run:
-          name: Archive Docker image
-          command: docker save -o image.tar $IMAGE_NAME
-      - persist_to_workspace:
-          root: .
-          paths:
-            - ./image.tar
-
-  publish-on-docker-hub:
-    executor: docker-publisher
-    steps:
-      - attach_workspace:
-          at: /tmp/workspace
-      - setup_remote_docker
+          name: Create a builder instance
+          command: docker buildx create --use
```

Comment:
This step is necessary for the multi-platform build to succeed. (Took me a while to get it right… On the bright side, I know at least that the arm64 image uses a base image of the correct platform; since the failure message complained about executable format :sweat_smile:)